css/styles.css

```diff
@@ -42,6 +42,21 @@ header {
   padding: 15px 0;
 }
 
+.btn {
+  background-color: #fff;
+  color: #396DF2;
+  padding: 10px 20px;
+  border: 1px solid #7F8CFF;
+  border-radius: 8px;
+  cursor: pointer;
+  font-size: 1em;
+}
+
+.btn:hover {
+  background-color: #6070FF;
+  color: #fff;
+}
+
```

Comment:
- Good job adding the hover effect, I suggest adding the active and enabled too as shown on the Figma design:
![Screenshot from 2023-03-24 19-06-37](https://user-images.githubusercontent.com/97834160/227593316-a32d28f9-619b-4583-ba76-6bf9df86c0a7.png)